src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/RunWorkflowTests.kt

```diff
@@ -45,8 +49,19 @@ class RunWorkflowTests : SeleniumTest()
         Assertions.assertThat(createButton.isEnabled).isTrue()
         Assertions.assertThat(createButton.text).isEqualTo("Create a blank workflow")
         createButton.click()
-
-        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("add-report-header")))
+        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("git-branch")))
+        val branchSelect = driver.findElement(By.id("git-branch"))
+        Assertions.assertThat(branchSelect.getAttribute("value")).isEqualTo("master")
+        val commitSelect = driver.findElement(By.id("git-commit"))
+        val commitValue = commitSelect.getAttribute("value")
+        Assertions.assertThat(commitValue).isNotBlank()
+
+        //Select a git branch
+        Select(branchSelect).selectByIndex(1)
+        Assertions.assertThat(branchSelect.getAttribute("value")).isEqualTo("other")
+        //Default commit value should update when new branch selected
+        wait.until(not(ExpectedConditions.attributeToBe(commitSelect, "value", commitValue)))
+        Assertions.assertThat(commitSelect.getAttribute("value")).isNotBlank()
     }
```

Comment:
Selenium test should assert buttons and all components display correctly.

Reply:
I'll add in a test for 'Refresh git', but don't think there's much point checking for the remaining placeholder components - will test the real ones in next branch.

Reply:
Also - I'm not sure it's completely true that Selenium tests should assert that all components display correctly - that should be mostly covered by the unit tests. Selenium tests should test interactivity and integration is working where that can't be fully tested in unit tests. We've tried to minimise how much we need to do in Selenium tests because they take so long to run! :laughing:

Reply:
I know, they also take long to debug.